api/src/main/java/marquez/api/OpenLineageResource.java

```diff
@@ -80,10 +91,48 @@ private int determineStatusCode(Throwable e) {
   @GET
   @Consumes(APPLICATION_JSON)
   @Produces(APPLICATION_JSON)
+  @Path("/lineage")
   public Response getLineage(
       @QueryParam("nodeId") @NotNull NodeId nodeId,
-      @QueryParam("depth") @DefaultValue(DEFAULT_DEPTH) int depth)
-      throws ExecutionException, InterruptedException {
+      @QueryParam("depth") @DefaultValue(DEFAULT_DEPTH) int depth) {
     return Response.ok(lineageService.lineage(nodeId, depth)).build();
   }
+
+  @Timed
+  @ResponseMetered
+  @ExceptionMetered
+  @GET
+  @Path("/events/lineage")
+  @Produces(APPLICATION_JSON)
+  public Response getLineageEvents(
+      @QueryParam("before") @DefaultValue("2030-01-01T00:00:00+00:00") ZonedDateTimeParam before,
```

Comment:
would be great if there was a way to make it `now`

Reply:
Unfortunately, I don't think this is possible: https://www.baeldung.com/java-annotation-attribute-value-restrictions

Also, `now` could potentially miss events if the Marquez server's clock is lagging - when the event time of some events is in the future.